.github/workflows/test-sqlite.yml

```diff
@@ -0,0 +1,24 @@
+name: test-sqlite
+
+on:
+  push:
+    branches: [ $default-branch ]
```

Comment:
Does this work here? I thought this was only for templates.

Do we need `${{ github.event.repository.default_branch }}` ?

Reply:
ooh it worked this time!

lmk if you have a shorter form of this.

Reply:
Yeah, I mean I just put in the specific name. It's not like default branch names change all that often anyway.